src/components/datepicker/datePicker.js

```diff
@@ -179,11 +202,23 @@
     this.ngModelCtrl = null;
 
     /** @type {HTMLInputElement} */
-    this.inputElement = $element[0].querySelector('input');
+    this.inputElement = $element[0].querySelector('input.begin');
```

Comment:
When you change the CSS class, this can just be `querySelector('.md-datepicker-start-date');`